code-challenges.test.js

```diff
@@ -44,11 +61,27 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
-
+//use a .push
 
 
 // b) Create the function that makes the test pass.
-
+describe("when an array of strings and numbers is called", ()=> {
+  it("returns an array of only odd numbers sorted from least to greatest", ()=> {
+    const sortArray(array){
+      let oddNums = [];
+      for(let i = 0; i < array.length; i++) {
+        if(array[i] % 2 !== 0) {
+          oddNums.push(array[i]);
+        }
+      }
+      oddNums = oddNums.sort((a,b)=> a - b);
+      array.concat(oddNums);
```

Comment:
looks like you don't need to concat the original array to the oddNums array because the end result of the function should only be odd numbers